examples/Avahi/main.cpp

```diff
@@ -0,0 +1,5 @@
+#include <avahi-client/client.h>
+#include <dns_sd.h>
```

Comment:
That's right. That's again the mix that my system provides, as the ubuntu-avahi-compat-libdns_sd installs it directly in usr/include/. I push the change.